examples/scenes/src/download/default_downloads.rs

```
@@ -39,5 +39,65 @@ pub(super) fn default_downloads() -> Vec<SVGDownload> {
             url: "https://upload.wikimedia.org/wikipedia/commons/5/58/Coat_of_arms_of_the_Kingdom_of_Yugoslavia.svg".to_string(),
             name: "Coat of Arms of the Kingdom of Yugoslavia".to_string()
         },
+        SVGDownload {
+            builtin:Some(BuiltinSvgProps {
+                info: "resvg",
+                license: "MIT",
+                expected_size: 383,
+            }),
+            url: "https://raw.githubusercontent.com/RazrFalcon/resvg-test-suite/master/tests/painting/stroke-dashoffset/default.svg".to_string(),
```

I'd consider using a fixed commit here, although knowing about an update is also good.

Maybe we need to just skip the specific item if the size doesn't match, rather than completely bailing out

---

If the size doesn't match the CLI already gives you the option to continue and ignore the error.

---

I just forgot I implemented that lol. This is a non-issue then